packages/labs/gen-wrapper-angular/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "private": true,
+  "name": "@lit-labs/gen-wrapper-angular",
+  "description": "Code generator for Angular wrappers for Lit components",
+  "version": "0.0.0",
+  "author": "Google LLC",
+  "license": "BSD-3-Clause",
+  "bugs": "https://github.com/lit/lit/issues",
+  "type": "module",
+  "main": "lib/index.js",
+  "repository": "lit/lit",
+  "scripts": {
+    "build": "wireit",
+    "test": "wireit",
+    "test:gen": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "command": "tsc --build --pretty",
+      "dependencies": [
+        "../gen-utils:build"
+      ],
+      "files": [
+        "src/**/*.ts",
+        "tsconfig.json"
+      ],
+      "output": [
+        "lib",
+        "index.{js,js.map,d.ts,d.ts.map}",
```

Comment:
Need a `"test"` here I think? It would explain why CI failed if you had a cache hit on build and the `test` folder wasn't restored. But there was only one commit afaict, so not sure.

Reply:
done